### PR TITLE
Templatetask

### PR DESCRIPTION
**Pull request by hannahgolightly**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -50,6 +51,11 @@ nav {
 
 }
 
+.top_nav_content {
+	float: right;
+
```
Comment (mgirar84) [inline style.css]: remove additional spaces.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+			</div>
+		</div>
+	</header>
+
+
+	<!-- about me section -->
+	<div id="about"></div>
+
+		<div class="container_main">
+			<div id="about_me_box">
```
Comment (mgirar84) [inline index.html]: can be changed to a class.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+						tincidunt id. Eu lobortis elementum nibh tellus molestie nunc non blandit. Aliquet nec
+						ullamcorper sit amet risus. Vitae tortor condimentum lacinia quis vel eros donec. Adipiscing
+						elit duis tristique sollicitudin nibh sit. Ac felis donec et odio. </p>
+				</div>
+				<img src=# class="img_me" alt="food" />
+			</div>
+		</div>
+
+
+		<!-- portfolio section -->
```
Comment (mgirar84) [inline index.html]: extra spacing

File: index.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+		<!-- footer section with copyright and link to go back to top of page -->
+		<footer>
+			<div class="container_main">
+				<p class="copyright">&copy; Hannah Golightly</p>
+				<a class="to_top" href="#home">Back to top</a>
+			</div>
+		</footer>
+
+
```
Comment (mgirar84) [inline index.html]: Check indentation

File: index.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+				<h2 class="logo_golightly">Golightly<span class="logo_dot">.</span></h2>
+			</a>
+			
+
+			<div class="top_nav_content">
+				<a href="#about">About Me</a>
+				<a href="#portfolio">My Work</a>
+				<a href="#contact">Contact</a>
+			</div>
+			</div>
```
Comment (mgirar84) [inline index.html]: remove indentation by one

File: style.css [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+/* portfolio section- these 3 boxes sit next to each other */
+.box1, .box2, .box3  {
+	padding: 0;
+	margin: 0;
+	width: 33.333%;
+	height: 350px;
+}
+
+.box1 {
+		background-color: #FFE2D1;
```
Comment (mgirar84) [inline style.css]: indentation

**Review by mgirar84 — CHANGES_REQUESTED** [review]
Suggested changes are mostly indentations and spacing.
Might worth asking someone else to check but looks good to me.

File: index.html (hunk 5 above)
Comment (hannahgolightly) [reply index.html]: done thanks

**Review by mgirar84 — APPROVED** [review]

**Review by mgirar84 — APPROVED** [review]

**Pull request merged**